ui/css/treeherder-navbar-panels.css

```diff
@@ -24,8 +24,6 @@
 .th-option-group {
     margin-right: 5px;
     border: 1px solid #101010;
-    -moz-border-radius: 5px;
```

Comment:
This `-moz` property wasn't being used, it is working fine with the generic non-moz border radius immediately below. Visible mostly in things like the current Filter panel container groups.